application/models/Permission.php

```diff
@@ -35,6 +35,9 @@ class Permission extends LSActiveRecord
     /* @var array[]|null The global base Permission LimeSurvey installation */
     protected static $aGlobalBasePermissions;
 
+/* @var interger| null the current userId for all permission check */
+    private static $currentUserId;
```

Comment:
Is this necesarry? Wouldn't this create problems as being duplicaed from Application?

Reply:
I don't remove it ?

Sorry … must be done